packages/playwright/src/common/ipc.ts

```diff
@@ -147,6 +148,14 @@ export function serializeConfig(config: FullConfigInternal, passCompilationCache
     configCLIOverrides: config.configCLIOverrides,
     compilationCache: passCompilationCache ? serializeCompilationCache() : undefined,
   };
+
+  try {
+    result.metadata = JSON.stringify(config.config.metadata);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error('Failed to serialize config metadata', error);
```

Comment:
Let's drop this error - no need to spam the user and it's ok if we failed. Otherwise we'd have to plumb it through the reporters api.